### PR TITLE
Fix naming in dnf5.spec

### PR DESCRIPTION
**Pull request by inknos**

### CODE REVIEW

File: dnf5.spec [hunk 1]
```diff
@@ -2,14 +2,35 @@
 %global project_version_minor 0
 %global project_version_patch 0
 
-Name:           libdnf5
+Name:           dnf5
 Version:        %{project_version_major}.%{project_version_minor}.%{project_version_patch}
 Release:        0~pre%{?dist}
 Summary:        Package management library
 License:        LGPLv2.1+
 URL:            https://github.com/rpm-software-management/libdnf
```
Comment (m-blaha) [inline dnf5.spec:10]: You need to change also repository URL.

File: dnf5.spec [hunk 2]
```diff
@@ -2,14 +2,35 @@
 %global project_version_minor 0
 %global project_version_patch 0
 
-Name:           libdnf5
+Name:           dnf5
 Version:        %{project_version_major}.%{project_version_minor}.%{project_version_patch}
 Release:        0~pre%{?dist}
 Summary:        Package management library
 License:        LGPLv2.1+
```
Comment (m-blaha) [inline dnf5.spec:9]: Also Summary and License should be from dnf5 subpackage:
```
Summary:        Command-line package manager
License:        GPLv2+
```

And to library subpackages (libdnf5, libdnf5-cli, -devel, and language bindings) should go `License: LGPLv2.1+`

File: dnf5.spec (hunk 1 above)
Comment (inknos) [reply dnf5.spec:10]: right, thanks

**Comment by m-blaha** [comment]
Thanks, LGTM.

**Pull request merged**